### ISSUE TITLE
authentication freezing on launch

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#3231]

**Comment by smashah** [comment]
Due to require module issue

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
try {
	execSync(cmd, {stdio: 'inherit'})
}  catch (error) {
    console.log(`Status Code: ${error.status} with '${error.message}'`);
 }
}
//set the version type
process.env.VERS = "patch"

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci $VERS`);

// types only package
exec('npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN')
exec(`npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN && npm run types:publish `);
exec(`git commit -a -m 'updated types-only package'`);
exec(`git push --force`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [4.71.15](https://github.com/open-wa/wa-automate-nodejs/compare/4.71.14...4.71.15) (2024-07-20)

- 🚑 atttempt auth freezing fix #3231 [`#3231`](https://github.com/open-wa/wa-automate-nodejs/issues/3231)
- 📝 docs update [`e8d7640`](https://github.com/open-wa/wa-automate-nodejs/commit/e8d76408910507d676ad7b6031c866d7173960d1)
- Release 4.71.15 [`d34a805`](https://github.com/open-wa/wa-automate-nodejs/commit/d34a80556105f59219d65b20faf11cc6a7a7cb87)
- 🥚 patches update [`92b00cc`](https://github.com/open-wa/wa-automate-nodejs/commit/92b00ccf39a96f6d8cbcf6dbe20d652612dc8c34)
- updated types-only package [`bad14c3`](https://github.com/open-wa/wa-automate-nodejs/commit/bad14c3aac1e9b2f6412ebe8321b45a570383ab9)
